Contrib/FreeWilson/test/test_freewilson.py

```diff
@@ -0,0 +1,63 @@
+import freewilson as fw
```

Comment:
Comment on the file in general:
Again, this isn't part of the core so it's not a deal breaker, but how is this supposed to get run? Is there a reason you didn't use python's unit test framework for it?

Reply:
The directions to run the tests are in the readme.